pkg/config/application.go

```diff
@@ -223,6 +223,7 @@ type PipelineStage struct {
 	Name    model.Stage
 	Desc    string
 	Timeout Duration
+	With    json.RawMessage
```

Comment:
Only this part is the only change that not under the pipedv1 directory. This is for packing stage configuration (not unmarshall) and pass it as is to the plugin.